### ISSUE TITLE
Group_concat fields not visible in Form View

### ISSUE THREAD

**Issue by balajiramanathan** [pawelsalawa/sqlitestudio#3764]
### Details
Any field that is created by using group_concat() is not visible in form view, but is visible in grid view

### Steps to reproduce
Create a view or query with one field being created by group_concat(), then view the contents of the view or query in grid view.  Switching to form view makes the group_concat field empty, and when you switch to grid view again, it shows NULL there.  Refreshing the view fills in the group_concat field in grid view again.

I use group_concat in the following way:  group_concat(MyTextField,char(13)).  This is for combining multiple text fields with a carriage return between them.

### Operating system
Windows 7

### SQLiteStudio version
3.31.1

**Comment by balajiramanathan** [comment]
Sometimes the form view shows the actual group_concat() function call instead of the data produced by the function call.  When I switch back to grid view after that, the data disappears and the group_concat() command shows up in the grid.
![Capture](https://user-images.githubusercontent.com/11827185/82130301-c5a0f900-978f-11ea-9f36-c7e82ffeeaa9.PNG)
![Capture1](https://user-images.githubusercontent.com/11827185/82130302-c76abc80-978f-11ea-8fc4-8af58bd69c87.PNG)

**Comment by pawelsalawa** [comment]
I cannot reproduce it, works for me every time. Can you provide minimal reproducible example of table + view? (as CREATE TABLE / CREATE VIEW statements).

**Comment by balajiramanathan** [comment]
I am not able to find a small test database in which this happens.  It is a problem in my production database, but I can't seem to construct a small example to reproduce the problem.

**Comment by d-01** [comment]
### Grid/Form view bug with `group_concat(X)` output

This bug occurs when `group_concat(X)` results in 100 character string or more.

### Steps to reproduce

1. Paste this SQL-query in SQL editor:

    ```sql
    WITH t1 (c1) AS (
        VALUES ('> 99 characters ------------------------------------------------------------------------------------')
    )
    SELECT group_concat(c1) as c2
    FROM t1
    ```
1. Execute (F9)
1. Switch to Form view
1. Switch to Grid view

*Expected*

Query result is displayed in Grid and Form view. Query result should be a string:
```
> 99 characters ------------------------------------------------------------------------------------
```

*Got*

This string is displayed instead of actual query result:
```
group_concat(c1)
```

### Demo

1. SQLiteStudio About window
2. Works as expected with string length=99
3. Bug occurs with string length=100

![2021-01-17_212212+03_0048](https://user-images.githubusercontent.com/45796739/104852205-8f151280-590a-11eb-9be2-aa30a04ea0d0.gif)

### Details

1. SQLiteStudio v3.2.1
2. Windows 7 x64

**Comment by tuffnatty** [comment]
Fixed since version 3.3.3 with commit 92b7789849ec12e0b8a2a19d5e6fc5116734deb0.
